js_modules/dagit/packages/ui/src/components/TokenizingField.tsx

```diff
@@ -429,7 +432,6 @@ export const TokenizingField: React.FC<TokenizingFieldProps> = ({
           ) : undefined
         }
       />
-      <StyledTagInput />
```

Comment:
Not sure how long this has been sitting here, but it's not supposed to be here and it causes `Popover2` warnings in console. 😆